uber/decorators.py

```diff
@@ -238,7 +237,11 @@ def with_restrictions(*args, **kwargs):
                 raise HTTPRedirect('../accounts/login?message=You+are+not+logged+in')
 
             else:
-                if not set(func.restricted).intersection(AdminAccount.access_set()):
```

Comment:
This is my fix for the issue I mentioned; basically we only consider ``REG_AT_CON`` when we're at the door.